### ISSUE TITLE
llama 3.2 loaded in the app but response (Referential Action). if we put second question then app crashes.

### ISSUE THREAD

**Issue by Deswaluday** [pytorch/executorch#6165]
### 🐛 Describe the bug

App loaded successfully using llama 3.2 1 b model in Samsung galaxy s24 ultra(12 gb Ram) .

giving response 
![image](https://github.com/user-attachments/assets/f8783757-126e-4519-822c-b6cad12d1e82)

and after i put next prompt to the app it will crash.

Below attached logs .

Please check the logs and suggest what i am doing wrong in this.

Also llava model is loading in this device but crash after putting the image .

Please provide a solution.

### Versions

![image](https://github.com/user-attachments/assets/40448891-7e88-4a1a-ab4d-eb129dd49830)

![image](https://github.com/user-attachments/assets/7d8513ba-fb98-4997-b2c8-5b1147b166d7)
[logs.txt](https://github.com/user-attachments/files/17341574/logs.txt)

**Comment by mergennachin** [comment]
Hi @Deswaluday, looks like it went over the max_seq_len

We should handle it gracefully regardless.

Can you increase "max_seq_length" when exporting via "export_llama" script?

**Comment by Riandy** [comment]
Also, you might be exporting the "base" model instead of the "instruct" version. Since the app is optimized for "chat" use cases, and it's equipped with the prompt format. We have seen some cases where base model  + instruct chat prompt can cause endless generation and incorrect response.

Tldr, export the instruct version of the model with longer seqlen for example 2048. And let us know if you are still seeing issues.

**Comment by Deswaluday** [comment]
Thanks this solution works for me..
